src/main/resources/db/changelog/alert/4.0.0/changelog-configuration-stored-procedures.xml

```diff
@@ -364,4 +364,26 @@
             $$;
         </createProcedure>
     </changeSet>
-</databaseChangeLog>
\ No newline at end of file
+    <!-- Create job migration procedure -->
+    <changeSet author="psantos" id="1545000000000-4">
+        <createProcedure
+                dbms="h2"
+                relativeToChangelogFile="true"
+                schemaName="ALERT"
+                procedureName="INSERT_FIELD_VALUE"
+        >
+            CREATE ALIAS INSERT_GLOBAL_FIELD_VALUE AS $$
+                java.lang.Void insertGlobalFieldValue(final java.sql.Connection connection, final java.lang.String descriptor_name, final java.lang.String fieldKey, final java.lang.String fieldValue) throws java.sql.SQLException {
```

Comment:
This is minor, but you're using both snake_case and camelCase for the method parameters.

Reply:
I'll fix it